### PR TITLE
Ma30 609

### PR DESCRIPTION
**Pull request by cah-venkatesh-manchala**

### CODE REVIEW

File: tests/http-input/http-input.test.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+const app = require("../../src/app");
+const send = require("../../src/send");
+const config = require("../../configs/mck-config.json");
+process.env.CONFIG = "mck-config";
+const route = require("../../src/http-input");
+jest.mock("../../src/send");
+
+describe("http-input", () => {
+  let server;
+  let payload = {
```
Comment (cah-rainy-jain) [inline tests/http-input/http-input.test.js:10]: Please update the tests wrt the schema

**Review by cah-josef-salyer — CHANGES_REQUESTED** [review]
Is there any way to break these changes up?  It seems like every file changed for just the http-input....

File: test-report/index.html [hunk 2]
```diff
@@ -0,0 +1,232 @@
+<html><head><meta charset="utf-8"/><title>Program Management</title><style type="text/css">html,
```
[the remark this answers is not in the document]
Comment (cah-venkatesh-manchala) [reply test-report/index.html:1]: I will add this file to .gitignore file

File: tests/load-patient-state/load-patient-state.test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+const config = require("../../configs/mck-config.json");
+
+describe("load-patient-state", () => {
+  let server;
+  beforeEach(() => {
+    server = app(route, config);
+  });
+  afterEach(() => jest.resetAllMocks());
+
+  it("should return 200 from post route", async () => {
```
[the remark this answers is not in the document]
Comment (cah-venkatesh-manchala) [reply tests/load-patient-state/load-patient-state.test.js:14]: Except http-input, all of the test classes are just present. so I just left it. The people who work in load-patient will rewrite these test cases.

File: index.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
 require("dotenv").config();
+const fs = require("fs");
 const fn = `./src/${process.env.FN}`;
+const schemaPath = `./src/${process.env.FN}/request.schema.json`;
```
Comment (cah-rainy-jain) [inline index.js:4]: I dont think we are using request.schema.json

**Review by JHock974 — APPROVED** [review]

**Review by cah-josef-salyer — APPROVED** [review]

**Pull request merged**